feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeState.kt

```diff
@@ -5,7 +5,7 @@ import com.epmedu.animeal.home.data.model.FeedingPoint
 import com.mapbox.maps.Style
 
 data class HomeState(
-    val currentLocation: MapLocation = MapLocation(),
+    val currentLocation: MapLocation = MapLocation.Initial,
```

Comment:
Why didn't you just change Initial to Tbilisi here?

Reply:
Ok, changed